src/components/MakeWidget/helpers/index.ts

```diff
@@ -1,6 +1,10 @@
+import { BigNumber } from "bignumber.js";
 import i18n from "i18next";
 
+import { getHumanReadableNumber } from "../../../helpers/getHumanReadableNumber";
+import stringToSignificantDecimals from "../../../helpers/stringToSignificantDecimals";
 import { TokenSelectModalTypes } from "../../../types/tokenSelectModalTypes";
+import { MakeWidgetState } from "../MakeWidget";
```

Comment:
On my end I see the error `Module '"../MakeWidget"' has no exported member 'MakeWidgetState'. Did you mean to use 'import MakeWidgetState from "../MakeWidget"' instead?`